spec/models/user_spec.rb

```diff
@@ -22,4 +27,14 @@
       expect(user.in_queue?(video)).to be true
     end
   end
+
+  describe '#gravatar' do
+    let(:user) { Fabricate(:user) }
+    it 'generates a gravitar url' do
+      expect(user.gravatar).to include("http://www.gravatar.com/avatar/")
```

Comment:
Yep: https://www.relishapp.com/rspec/rspec-expectations/docs/built-in-matchers/include-matcher